### PR TITLE
Add p2p support

### PR DESCRIPTION
**Pull request by HileQAQ**
Signed-off-by: Haoqi Miao <miaohaoqi.mhq@alibaba-inc.com>

### CODE REVIEW

**Comment by BigVan** [comment]
LGTM #115

**Comment by liulanzheng** [comment]
please add a config description here: https://github.com/containerd/overlaybd/blob/main/README.md#configuration

File: README.md [hunk 1]
```diff
@@ -121,13 +121,17 @@ Default configure file `overlaybd.json` is installed to `/etc/overlaybd/`.
         "delayExtra": 30,
         "maxMBps": 100
     },
+    "p2pConfig": {
+        "enable": false,
+        "address": "http://localhost:9731/accelerator"
```
Comment (liulanzheng) [inline README.md:126]: remove http:// prefix?

Comment (HileQAQ) [reply README.md:126]: It makes no difference.

**Pull request merged**